src/actions/utilActions.js

```diff
@@ -172,20 +172,35 @@ export function updateAvatar(user, form_data) {
   };
 }
 
+export function deleteUser(user, dispatch) {
+  const username = user.username;
+  Server.delete(`delete/user/${user.id}/`, user)
+    .then(response => {
+      dispatch(fetchUserList());
+      showNotification({
+        message: i18n.t("toasts.deleteuser", { username: username }),
+        title: i18n.t("toasts.deleteusertitle"),
+        color: "teal",
+      });
+    })
+    .catch(error => {
+      console.error(error);
+      dispatch({ type: "DELETE_USER_REJECTED", payload: error });
+    });
+}
+
```

Comment:
I see you've also put this in the RTK api deffinition. That also generats functions for non-react-functions use. Do you still need this then?